src/modules/actions/getFilms.js

```diff
@@ -0,0 +1,18 @@
+import { GET_FILMS, API_KEY } from '../constants/constants';
+import getGenre from './getGenre';
+
+const getFilms = () => {
+  return async (dispatch) => {
+    await fetch(`https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`)
```

Comment:
plz move urls to the config file
and split it by separate constants like
```
const BASE_URL = 'https://api.themoviedb.org/3'
const categories ={
popular: 'popular',
....
}
```
then join to url 
`${BASE_URL }/movie/${categories.popular}/ .... and query string here ....` it will be needed for feature task - react router

Reply:
the same here